.editorconfig

```diff
@@ -61,6 +61,13 @@ dotnet_style_coalesce_expression = true:suggestion
 dotnet_style_null_propagation = true:suggestion
 dotnet_style_explicit_tuple_names = true:suggestion
 
+# Whitespace options
+dotnet_style_allow_multiple_blank_lines_experimental = false:warning
+dotnet_diagnostic.IDE2000.severity = warning
+
+dotnet_style_allow_statement_immediately_after_block_experimental = true:none
+dotnet_diagnostic.IDE2003.severity = none
```

Comment:
would like to set this to 'false'.  but will come in a followup PR.

Reply:
Can we set it to `false:suggestion` for this PR, and just not fix the current reports yet?